ctenex/domain/contract_codes.py

```diff
@@ -0,0 +1,5 @@
+from enum import StrEnum
+
+
+class ContractCode(StrEnum):
+    MAR_100_MW = "MAR_100_MW"
```

Comment:
Yea, not bad. Shall we add a few more, so it's not just this one guy? 
The structure of contract codes are up to the exchange, or the trading company (for OTC) to determine - but they're typically things like "UK-BL-MAR-25" for a March 25 delivery of baseload (i.e. around the clock) UK Power. Or "NBP-Q4-25" for UK Natural Gas delivery in Q4 2025 traded on the National Balancing Point. 

The quantity is not typically in the identifier. Standard contracts on exchanges have a predefined lot size (e.g., 1 MW for power delivered over each hour of the contract period), so you won’t see the quantity embedded in the ticker.